games/a.yaml

```
@@ -255,6 +255,28 @@
   type: remake
   updated: 2016-03-08
 
+- name: Ambermoon.net
+  type: remake
+  originals:
+    - Ambermoon
+  repo: https://github.com/Pyrdacor/Ambermoon.net
+  development: very active
+  status: playable
+  lang:
+    - 'C#'
+  license:
+    - GPL3
+  content: free
```

```suggestion
  content: commercial
```
since abandonware isn't a license